apps/src/tutorialExplorer/tutorialExplorer.js

```diff
@@ -423,9 +423,52 @@ function getFilters({robotics, mobile}) {
   return {filters, initialFilters, hideFilters};
 }
 
+function getUrlParameters(filters) {
+  let providedParameters = null;
+
+  const parameters = decodeURIComponent(window.location.search.substring(1)).split('&');
```

Comment:
Elsewhere we use query-string to access query params (https://github.com/sindresorhus/query-string). Would be good to use the same thing here.